admin_assignments_dates.php

```diff
@@ -1,517 +1,485 @@
+	<th scope="col"> M5 DL </th>
+     <th scope="col"> M6 Issue </th>
+	<th scope="col"> M6 DL </th>
+	<th scope="col"> M7 Issue </th>
+	<th scope="col"> M7 DL </th>
+     <th scope="col"> M8 Issue </th>
+	<th scope="col"> M8 DL </th>
+	<th scope="col"> M9 Issue </th>
+	<th scope="col"> M9 DL </th>
+	<th scope="col"> M10 Issue </th>
+	<th scope="col"> M10 DL </th>
+	<th scope="col" >Edit</th>
+	<th class="hidden" scope="col" >Delete</th>	
+		
+     
+    </tr>
+  </thead>
+  <tbody>
+	  <?php 
+ 		while ($row= $result->fetch_assoc()):  ?>  
+	  
+	<tr id="row_<?php echo $row['assignmnt_id']; ?>">
+		<td><?php echo $row['assignmnt_id']; ?></td>
+		<td><?php echo $row['batch_id']; ?></td>
+		<td><button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#assignmentDetails_<?php echo $row['assignmnt_id']; ?>"><i class="fa fa-plus"></i></button></td>
+		<td><button type="button" class="btn btn-success btn-xs editbtn"><i class="fa fa-cog" aria-hidden="true"></i></button></td>
+		<td><button type="button" class="btn btn-danger btn-xs deletebtn" data-id="<?php echo $row['assignmnt_id']; ?>"><i class="fa fa-trash"></i></button></td>
+	</tr>
+	<tr id="assignmentDetails_<?php echo $row['assignmnt_id']; ?>" class="collapse">
+		<td colspan="4">
+			<table class="table-condensed">
+				<thead>
+					<tr>
+						<th>Module Issued</th>
+						<th>Module Deadline</th>
+					</tr>
+				</thead>
+				<tbody>
+					<?php for ($i = 1; $i <= 10; $i++) { ?>
+						<?php if (!empty($row["module_{$i}_issued"]) && !empty($row["module_{$i}_deadl"])) { ?>
+							<tr>
+								<td><?php echo $row["module_{$i}_issued"]; ?></td>
+								<td><?php echo $row["module_{$i}_deadl"]; ?></td>
+							</tr>
+						<?php } ?>
+					<?php } ?>
+				</tbody>
+			</table>
+		</td>
+	</tr>
+   
+	  
+  </tbody>
+
+
+<?php endwhile; ?>
+</table>
+	<?php
+ 	 function pre_r( $array ){
+ 	 	echo '<pre>';
+ 	 	print_r($array);
+ 	 	echo '</pre>';
+
+ 	 }
+ 	 ?>
+		  
+
+		  </div>
+  
+		  </div>
+		  
+		  <div class="section3">
+		  
+		 <p  align="center" style="color:  white; padding-top: 30px; font-size:20px " >© 2021 MAY / GREENDALE COLLEGE </p>
+ 
+		  </div>
+	
+	  </div>
+ 
+ 
+
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="js/bootstrap.min.js"></script>
+
+
+<script>
+$(document).ready(function() {
+  $('.editbtn').on('click', function() {
+	$('#editmodal').modal('show');
+	var $tr = $(this).closest('tr');
+	var data = $tr.children("td").map(function() {
+	  return $(this).text();
+	}).get();
+	console.log(data);
+	for (var i = 1; i <= 10; i++) {
+	  $('#module_' + i + '_issued').val(data[2 * i]);
+	  $('#module_' + i + '_deadl').val(data[2 * i + 1]);
+	}
+	$('#update_id, #assignmnt_id, #batch_id').val(data[0]); 
+  });
+});

```

Comment:
This version uses a for loop to iterate over the 10 sets of module fields and sets their values dynamically with string concatenation. It also chains together the assignment of `update_id , ``assignmnt_id `, and` batch_id` with `jQuery's `multiple selector syntax to make the code more compact